test/funnels/callees/SwapperCalleeUniV3.t.sol

```diff
@@ -30,6 +30,8 @@ contract SwapperCalleeUniV3Test is DssTest {
     }
 
     function testConstructor() public {
+        vm.expectEmit(true, true, true, true);
+        emit Rely(address(this));
```

Comment:
```suggestion
```
This should be reverted, there is not `wards` structure in this contract.

Reply:
Yes, saw it at the same time, fixed.